w2w/w2w.py

```diff
@@ -40,6 +39,21 @@
     import importlib_resources
 
 
+# WRF version dict
+WRF_VERSIONS_DICT = {
+    'v4.3': {'ADD_LCZ_INT': 30, 'NUM_LAND_CAT': 41},
+    'v4.3.1': {'ADD_LCZ_INT': 30, 'NUM_LAND_CAT': 41},
+    'v4.3.2': {'ADD_LCZ_INT': 30, 'NUM_LAND_CAT': 41},
+    'v4.3.3': {'ADD_LCZ_INT': 30, 'NUM_LAND_CAT': 41},
+    'v4.4': {'ADD_LCZ_INT': 30, 'NUM_LAND_CAT': 41},
+    'v4.4.1': {'ADD_LCZ_INT': 30, 'NUM_LAND_CAT': 41},
+    'v4.4.2': {'ADD_LCZ_INT': 50, 'NUM_LAND_CAT': 61},
+    'v4.5': {'ADD_LCZ_INT': 50, 'NUM_LAND_CAT': 61},
+    'v4.5.1': {'ADD_LCZ_INT': 50, 'NUM_LAND_CAT': 61},
+    'v4.5.2': {'ADD_LCZ_INT': 50, 'NUM_LAND_CAT': 61},
+}
+
+
```

Comment:
let's have the dict in a single location and add new versions only here. This automatically populates the argument validation.

Reply:
I missed this comment/question. What do you mean by this? You want to have this in a separate file? Or ...? 
It is probably not a bad idea, since this dict probably needs updates whenever a new WRF version is released.
Even though the next WRF release will have W2W integrated into the WPS/WRF system itself ...

Reply:
Nah, just as a module-level constant as it currently is so in **one** place only. This way, changing things in that dict will automatically change e.g. the help texts, the valid choices etc.